modules/javafx.web/src/test/java/test/javafx/scene/web/LocalStorageTest.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package test.javafx.scene.web;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static java.lang.String.format;
+import java.io.RandomAccessFile;
+import java.nio.channels.FileLock;
+import java.util.concurrent.FutureTask;
+import java.io.File;
+import java.io.IOException;
+import java.util.Random;
+
+
+import javafx.scene.web.WebEngineShim;
+import javafx.concurrent.Worker;
+import javafx.scene.web.WebView;
+import javafx.scene.web.WebEngine;
+
+
+public class LocalStorageTest extends TestBase {
+
+    private static final File LOCAL_STORAGE_DIR = new File("LocalStorageDir");
+    private static final File PRE_LOCKED = new File("zoo_local_storage");
```

Comment:
This looks like a copy / paste from `UserDataDirectoryTest`, and is unused (and not needed) in this test.

Reply:
LOCAL_STORAGE_DIR is used by web engine , to store local data in this case

Reply:
private static final File LOCAL_STORAGE_DIR = new File("LocalStorageDir");  is required, and i agree
    not require for private static final File PRE_LOCKED = new File("zoo_local_storage");